### PR TITLE
Display error message and stay on same page when line is not created

### PR DESCRIPTION
**Pull request by knuepwebdev**
https://www.pivotaltracker.com/story/show/112162881

### CODE REVIEW

File: app/controllers/lines_controller.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ def create
     if @line.save
       redirect_to root_url, notice: "Haiku line created!"
     else
+      flash[:error] = 'Error. You must be logged in and line cannot be blank'
```
Comment (kangkyu) [inline app/controllers/lines_controller.rb]: For not logged in, we have a flash message [here](https://github.com/charliemcelfresh/haikus/blob/master/app/controllers/application_controller.rb#L17) already

**Comment by kangkyu** [comment]
Can we use `@line.errors` instead of `flash[:error]`?

File: spec/requests/lines_spec.rb [hunk 2]
```diff
@@ -40,6 +40,16 @@
...
       end
+
+      it 'should stay on the new line page when a line is not created' do
+        post "/haikus/#{haiku.id}/lines", "line" => { "content"=> nil }
+        expect(response).to render_template(:new)
+      end
+
+      it 'should display an error message' do
+        post "/haikus/#{haiku.id}/lines", "line" => { "content"=> nil }
+        expect(response.body).to include("Content can&#39;t be blank")
```
Comment (thomas-jung) [inline spec/requests/lines_spec.rb]: I am curious why you chose to type your apostrophe this way.

**Comment by thomas-jung** [comment]
MIke this looks good :+1:

File: spec/requests/lines_spec.rb (hunk 2 above)
Comment (knuepwebdev) [inline spec/requests/lines_spec.rb]: When I write the assertion like this then the test fails:
expect(response.body).to include("Content can't be blank")

I know this code is pretty ugly - the specs are passing though

**Comment by knuepwebdev** [comment]
Thanks Thomas!

**Pull request merged**